src/printer.rs

```diff
@@ -107,17 +110,57 @@ impl<'a, T: Read> BinaryGuard<'a, T> {
     }
 }
 
+pub struct FormatOptions {
```

Comment:
All of our CLI-related data types live in `src/cli.rs`. Can you move this struct to that file as well?